bin/shared_linode.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+if [ -z "$LINODE_TOKEN" ]
```

Comment:
I will fetch the value from the vault, but can you add the linode test to the GitHub workflow file? Let me know if you need help with this.

Reply:
The secret has been aded as LINODE_TOKEN